bundle/Command/LegacySrcSymlinkCommand.php

```diff
@@ -0,0 +1,150 @@
+any design/settings project files, and symlinks these into <info>ezpublish_legacy/</info> which is installed by composer.
+
+<comment>NOTE: Look towards 'ezpublish:legacybundles:install_extensions' command for how you handle extensions.</comment>
+EOT
+            );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $srcArg = rtrim($input->getArgument('src'), '/');
+        $create = (bool)$input->getOption('create');
+        $force = (bool)$input->getOption('force');
+
+        /**
+         * @var \Symfony\Component\Filesystem\Filesystem
+         */
+        $filesystem = $this->getContainer()->get('filesystem');
+
+        if (!$filesystem->exists($srcArg)) {
+            if ($create) {
+                $this->createSrcFolderStructure($filesystem, $srcArg);
+                $output->writeln("<comment>Empty legacy src folder was created in '$srcArg'.</comment>");
+                // No break, continue so setup symlink settings/override
+            } else {
+                throw new \InvalidArgumentException(sprintf('The src directory "%s" does not exist.', $srcArg));
+            }
+        }
+
+        $symlinkFolderStr = implode(' ,', $this->linkSrcFolders($filesystem, $srcArg, $force));
+
+        if ($symlinkFolderStr) {
+            $output->writeln("The following folders where symlinked: '$symlinkFolderStr'.");
+        } else {
+            $output->writeln('No folders where symlinked, use force option if they need to be re-created.');
+        }
+
+        $output->writeln(<<<EOT
+
+If you create or move additional design or siteaccess folders to '$srcArg' from previous install, then
+re-run <info>ezpublish:legacy:symlink_src</info> to setup symlinks to eZ Publish legacy folder for them also.
+
+EOT
+);
+    }
+
+    /**
+     * Create legacy src folder structure.
+     *
+     * src/legacy_files:
+     * - design
+     * - settings:
+     *   - override
+     *   - siteaccess
+     *
+     * @param Filesystem $filesystem
+     * @param string $srcArg
+     */
+    protected function createSrcFolderStructure(Filesystem $filesystem, $srcArg)
+    {
+        $filesystem->mkdir([
+            $srcArg,
+            "$srcArg/design",
+            "$srcArg/settings",
+            "$srcArg/settings/override",
+            "$srcArg/settings/siteaccess",
+        ]);
+    }
+
+    /**
+     * Setup symlinks for legacy settings/design files within eZ Publish legacy folder.
+     *
+     * @param Filesystem $filesystem
+     * @param string $srcArg
+     * @param bool $force
+     *
+     * @return array
+     */
+    protected function linkSrcFolders(Filesystem $filesystem, $srcArg, $force)
+    {
+        $symlinks = [];
+        $legacyRootDir = rtrim($this->getContainer()->getParameter('ezpublish_legacy.root_dir'), '/');
+
+        if ($force || !$filesystem->exists("$legacyRootDir/settings/override")) {
+            $filesystem->symlink(
```

Comment:
Should probably check if `"$srcArg/settings/override"` exists before symlinking.

Reply:
true, same for the rest